services/GoogleAdSense.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "Google AdSense",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://www.google.com/adsense/tc/2018/UnitedKingdom.html",
+      "select": "body"
+    },
+    "Program Policies": {
+      "fetch": "https://support.google.com/adsense/answer/48182",
+      "select": ".article-container",
+      "remove": ".action-button, .zippy"
+    },
+    "Parent Organization Privacy Policy": {
+      "fetch": "https://policies.google.com/privacy?hl=en-US",
+      "filter": [
+        "removeUTMfromUrls"
+      ],
+      "select": "div[role=article]"
+    },
+    "Brand Guidelines": {
+      "fetch": "https://support.google.com/adsense/answer/6370057",
+      "select": ".article-container"
+    },
+    "Quality Guidelines": {
+      "fetch": "https://support.google.com/webmasters/answer/35769",
```

Comment:
@MattiSG In this document the version generates an empty link to the intro video (`[](https://www.youtube.com/watch?v=yFxNda5Z4eE)`). Should we keep this or remove it, what do you think?

Reply:
As discussed: I don't have an answer for that as the provider of _CGUs_. I believe it is up to the contributor to decide if the intro video is meaningful (in which case it might be worth adding a filter to add a textual content to the link), useless (in which case it might be worth adding a filter to remove the link), or if they don't care 🙂

Reply:
Having quickly watched videos, I don't think they are very meaningful, but I'm not sure if we gain anything by removing them so I'm neutral about both decisions.